JSDemos/Demos/Form/ColumnsAdaptability/description.md

```diff
@@ -1 +1,21 @@
-The Form component can use different layouts depending on&nbsp;the screen width. The dependency between column count and the screen width is&nbsp;specified using the **colCountByScreen** property, which has a&nbsp;higher priority than the **colCount** and the **minColWidth** properties. In&nbsp;this demo, the form is&nbsp;displayed using a&nbsp;4-column layout on&nbsp;wide screens and a&nbsp;2-column layout on&nbsp;narrow screens. Switch between the landscape and portrait layouts to&nbsp;see how the form looks on&nbsp;wide and narrow screens.
\ No newline at end of file
+The Form component can use different layouts depending on the screen width. Click one of the buttons on the right from the demo title to switch between horizontal and vertical orientations. The number of columns in the layout can be specified in one of the following manners:
+
+**Calculate the Number of Columns Automatically**         
+
+In this case, the Form calculates the number of columns based on the screen width. To configure this behavior, follow these steps:
+
+1. Assign *"auto"* to the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property to make the number of columns adapt to any screen size. 
+
+1. Use the [minColWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#minColWidth) property to specify the minimum column width.
+
+In this demo, the number of columns is calculated automatically when the check box under the Form is clear.
+
+**Predefine the Number of Columns for Each Screen Size**       
+
+In this case, screen sizes are classified into one of the following categories called "size qualifiers": extra small, small, medium, or large. You specify the number of columns for each size qualifier. Follow the steps below to configure this behavior:
+
+1. Use the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) function to map screen widths to size qualifiers. In this demo, screens narrower than 720 pixels are classified as small; other screens are considered medium.
+
+1. Specify the [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCountByScreen) property to define how many columns a Form should contain depending on the available screen qualifiers. 
+
+In this demo, select the check box under the Form to enable this configuration.
```

Comment:
Does this checkbox have a name? I would rather say:
> Check the `CheckboxName` option under the Form to enable this configuration.

Reply:
No, it does not: 
![image](https://user-images.githubusercontent.com/33040934/114701744-e2410a80-9d2b-11eb-9259-04eb45e526e6.png)

Reply:
A pity. Why did I ask this in the first place? Because you said "select the check box", but it's equally strange to say "check the check box". From what I gather, this option is initially disabled. I can offer you "mark" instead of "select" as a possible alternative. Like this:
```suggestion
In this demo, mark the check box under the Form to enable this configuration.
```

Reply:
This makes sense